WordPress/gutenberg
# Added toolbar dropdown action to set any image as feature image

## What?
- if the feature image is not set for the post then if someone checked the toggle control of the image to set it as a feature image then it will be set as a feature image for the post.

## Why?
fixes #65889 

## How?
- added toggle control in inspector control to `Set this image as feature image`.
- if `isFeatureImage` attribute is set & post doesn't have a feature image then it will be set as a feature image for the post.

## Testing Instructions
- create a new post.
- add image block, and check `Set this image as feature image` toogle control of settings to set this image as feature image.
- check feature image of post.

## Screenshots or screencast 

https://github.com/user-attachments/assets/4576c7c0-61df-43fd-a062-c47a6029bbe5
